pkgs/shells/nix-your-shell/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "nix-your-shell";
+  version = "v1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "MercuryTechnologies";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-kdZFwMHatnhdXGSIItuE3g27qqUKqT/Hkbz13Ba5eq4=";
+  };
+
+  cargoSha256 = "sha256-dEEv0KVTPNB55ZGTsz2VpRHtmagbakd6Z4lmWab4lqw=";
+
+  meta = {
+    description = "A `nix` and `nix-shell` wrapper for shells other than `bash`";
+    homepage = "https://github.com/MercuryTechnologies/nix-your-shell";
+    license = [lib.licenses.mit];
+    maintainers = [lib.maintainers._9999years];
```

Comment:
```suggestion
  meta = with lib; {
    description = "A `nix` and `nix-shell` wrapper for shells other than `bash`";
    homepage = "https://github.com/MercuryTechnologies/nix-your-shell";
    license = licenses.mit;
    maintainers = with maintainers; [ _9999years ];
```

Reply:
Done.

Reply:
I think you forgot to commit the change

Reply:
Check the diff. I didn't use the 'commit suggestion' button but I did make the change.

Reply:
Not quite what I suggested but this will do